### PR TITLE
Minor touches

### PR DESCRIPTION
**Pull request by martinfleis**
Minor touches:

- removed splot and replaced it with W.plot that allows us to control the styling and removes one dependency (why are there two different methods of plotting weights? shouldn't libpysal import the method from splot??)
- replaced `sw_high` with `libpysal.weights.higher_order` as that now has the same options and I'd like to deprecate `sw_high`
- minor change of tqdm to use the notebook widget
- nicer examples on tessellation
- expanded installation instructions if folks want to run locally
- and some tiny bits

### CODE REVIEW

File: README.md
Comment (jGaboardi) [inline README.md]: delete the (s) from *notebooks*

**Comment by jGaboardi** [comment]
@martinfleis I am working on some more touchup on your notebook locally, so don't merge this yet.

**Comment by jGaboardi** [comment]
@martinfleis can you "allow commits from maintainers" here?

**Comment by martinfleis** [comment]
You have commit rights to the repo, don't you?

**Comment by jGaboardi** [comment]
Yeah... I must be doing something wrong. I always forget how to push up commits to another PR.

**Comment by martinfleis** [comment]
just push to the mf_edits branch. I don't have a fork so it should be okay

**Comment by jGaboardi** [comment]
* Nice work for the `tqdm` adjustments.
* Some adjustments/musings:
  * minor formatting and variable declaration changes for the plotting chunks
  * curious why the buildings + weights plot in cell 38 takes so long (≈1 minute)...
  * added back the `warning` filter for the `clustergram` plot (cell 67)
  * removes the `splot` reference

**Review by jGaboardi — APPROVED** [review]

**Comment by martinfleis** [comment]
> curious why the buildings + weights plot in cell 38 takes so long (≈1 minute)

Because the code is inefficient. I may actually leave it out... The one on the toy example should be enough.

**Comment by martinfleis** [comment]
Done. I have also cleared the outputs to have it ready for the tutorial and to avoid it being 90mb large.

**Pull request merged**
